lib/components/viewers/related-panel.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react'
+
+class RelatedPanel extends Component {
+  render () {
+    const { children, count, expanded, title } = this.props
```

Comment:
Should we stick to using functional components if all a component does is render?